src/sentry/static/sentry/app/views/organizationDetails/index.jsx

```diff
@@ -161,8 +175,14 @@ export default class OrganizationDetails extends Component {
   render() {
     return (
       <OrganizationContext includeSidebar useLastOrganization {...this.props}>
-        <OrganizationDetailsBody>{this.props.children}</OrganizationDetailsBody>
+        <OrganizationDetailsBody {...this.props}>
```

Comment:
Ah I mean we should destructure `detailed` from `this.props` and forward the rest to `OrganizationContext`

So it'd be something like...

```
const {detailed, ...props} = this.props;
<OrganizationContext includeSidebar useLastOrganization {...props}>
        <OrganizationDetailsBody detailed={detailed}>{this.props.children}</OrganizationDetailsBody>	       
``` 

This way, as a reader, I know that `OrganizationDetailsBody` requires `detailed` (and `children`) and `OrganizationContext` doesn't (and forward the rest of the props to `OrganizationContext`

Reply:
oooh I understand what you are saying, but I think organizationContext needs detailed because it is the component that ends up making the request to the organization details endpoint, so I'm not sure if we can destructure it like this.